modules/olm-mirroring-catalog-airgapped.adoc

```diff
@@ -19,6 +19,7 @@ If your mirror registry is on a completely disconnected, or airgapped, host, tak
 
 . Run the following command on your workstation with unrestricted network access to mirror the content to local files:
 +
+--
```

Comment:
This is not part of the PR, so I think it's optional to fix: because this multi-line codeblock includes callouts, you should comment the callouts so that the formatting is correct (https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#callouts-in-code-blocks).

For example:

```
$ oc adm catalog mirror \
    <index_image> \// <1>
    file:///local/index \// <2>
    -a ${REG_CREDS} \// <3>
    --insecure \// <4>
    --index-filter-by-os='<platform>/<arch>' <5>
```